ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main.yaml

```diff
@@ -92,6 +96,11 @@ tests:
   secret:
     mount_path: /tmp/secret
     name: assisted-image-service-codecov-token
+- as: e2e-metal-assisted
```

Comment:
I'm afraid it will have no effect without moving the dependency around (meaning, with this exact change it will only test master branch of image-service):
https://github.com/openshift/release/blob/29112064321b0bc26611ef7aba5a925fbe45ee18/ci-operator/step-registry/assisted/baremetal/setup/assisted-baremetal-setup-ref.yaml#L13-L29